app/policies/presentation_policy.rb

```diff
@@ -6,4 +6,12 @@ def create?
   def new?
     create?
   end
+
+  def update?
+    person.present? && (person.admin? || person.presentations.include?(record))
```

Comment:
Yeah the reason I put in the `.present?` call is because since this method ends in a ? it is implied that it returns a boolean. If I went `nil && true` I would get back `nil`. The same sort of problem arises with the guard operator.

```
!!(person&.admin? || person&.presentations&.include?(record))
```

Would be the outcome. I would have to swap the `nil` value to a false value. I also can't do a `!.nil?` check because that would error on false values. I've changed it to be the way I listed it above because I understand the importance of not erroring on empty arrays or strings even though I don't think our `current_person` would ever be set to one of those things but I do think that this solution is a little bit cryptic. 

I sent it by Nat this morning and she had a tough time reading it so I feel like if others have trouble reading it then it isn't the best.